### DISCUSSION TITLE
Any future plan for Cross platform navigation?

### DISCUSSION THREAD

**Discussion by duongphuhiep** [AvaloniaUI/Avalonia#16888] [Q&A] [answered]
Hello,

Avalonia supports Mobile platforms now. However, a big missing part is "Navigation". I means:

* Navigation History (Mobile + Browser with "push(route)", "replace(route)").
* "Virtual" back button (which bind to the Browser Back button AND Android physical Back button).
* Deep link.

AFAIK: Flutter, React-native, Maui.. all supports "Navigation" as a core component (they just use different jargons)

Does Avalonia have any plan to make "Navigation" as a core component? if yes then is it an important priority or was it a "Nice to have" feature we might plan for it in Avalonia 14.x? 

Note: I'm aware of some community initiative:

* <https://github.com/AvaloniaInside/Shell>
* <https://github.com/sandreas/Avalonia.SimpleRouter>

**Comment by MikeCodesDotNET** [comment, marked as answer]
I'm on vacation so will keep this short.  

Yes.
